doc/user/fabricd.rst

```diff
@@ -57,14 +57,27 @@ in the configuration:
 
    Configure the authentication password for a domain, as clear text or md5 one.
 
+.. index:: attached-bit [receive | send] [ignore]
```

Comment:
Fix your commit message to `doc:...`

The command format is `attached-bit [receive ignore|send]` correct?

Reply:
ok

Reply:
You said ok.. but the commit message is the same?

Reply:
I thought you meant to update the documentation.   I modified doc/user/isis.rst.  Was there something else I was supposed to modify?

Reply:
sorry I modified /doc/user/isisd.rst but forgot to also do fabricd.rst.   Just pushed the change